src/features/MenuPage/MenuCourses/Desserts/Desserts.js

```diff
@@ -0,0 +1,40 @@
+import {useEffect, useState} from "react";
+import MenuItem from "../MenuItem";
+
+const Desserts = () => {
+
+    const [desserts, setDesserts] = useState([])
+
+    const getDesserts = async () => {
+        const data = await fetch('http://localhost:3001/dishes/desserts')
+        return await data.json()
+    }
+
+    useEffect(() => {
+        getDesserts()
+            .then(json => {
+                if (json.success) {
+                    setDesserts(json.data)
+                }
+            })
+    },[])
+
+    const displayDesserts = () => {
+        if (desserts.length > 0) {
+            return desserts.map(dessert => {
+                return <MenuItem key={dessert._id} name={dessert.name} description={dessert.description}
+                                 price={dessert.price} image={dessert.imageURL}/>
+            })
+        }
+        else return null
```

Comment:
Would be nice to show an error message here - maybe some kind of generic error component that you can populate with a specific message